### PR TITLE
React Native SDK

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

- React Native SDK

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- https://github.com/appwrite/sdk-generator/pull/794

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: docs/sdks/react-native/GETTING_STARTED.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+
+## Getting Started
+
+### Add your Platform
+For you to init your SDK and interact with Appwrite services you need to add a web platform to your project. To add a new platform, go to your Appwrite console, choose the project you created in the step before and click the 'Add Platform' button.
+
+From the options, choose to add a **Web** platform and add your client app hostname. By adding your hostname to your project platform you are allowing cross-domain communication between your project and the Appwrite API.
```
Comment (gewenyu99) [inline docs/sdks/react-native/GETTING_STARTED.md:7]:
```suggestion
If this is your first time using Appwrite, create an account and create your first project.

Then, under **Add a platform**, add a **Android app** or a **Apple app**. You can skip optional steps.

#### iOS steps
Add your app **name** and **Bundle ID**. You can find your **Bundle Identifier** in the **General** tab for your app's primary target in XCode.

#### Android steps
Add your app's **name** and **package name**, Your package name is generally the **applicationId** in your app-level [build.gradle](https://github.com/appwrite/playground-for-flutter/blob/master/android/app/build.gradle#L41) file.
```

File: docs/sdks/react-native/GETTING_STARTED.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+```js
+import { Client } from 'react-native-appwrite';
+// Init your Web SDK
+const client = new Client();
+
+client
+    .setEndpoint('http://localhost/v1') // Your Appwrite Endpoint
+    .setProject('455x34dfkj') // Your project ID
+    .setPlatform('com.example.myappwriteapp') // YOUR application ID
```
Comment (gewenyu99) [inline docs/sdks/react-native/GETTING_STARTED.md:31]:
```suggestion
    .setPlatform('com.example.myappwriteapp') // Your application ID or bundle ID.
```

File: docs/sdks/react-native/GETTING_STARTED.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    .then(function (response) {
+        console.log(response);
+    }, function (error) {
+        console.log(error);
+    });
+```
+
+### Learn more
+You can use the following resources to learn more and get help
+- 🚀 [Getting Started Tutorial](https://appwrite.io/docs/getting-started-for-react-native)
```
Comment (gewenyu99) [inline docs/sdks/react-native/GETTING_STARTED.md:76]:
```suggestion
- 🚀 [Getting Started Tutorial](https://appwrite.io/docs/quick-starts/react-native)```

**Review by gewenyu99 — CHANGES_REQUESTED** [review]

**Review by abnegate — APPROVED** [review]

**Review by stnguyen90 — APPROVED** [review]

File: app/config/platforms.php [hunk 4]
```diff
@@ -135,6 +135,24 @@
...
                 ],
             ],
+            [
+                'key' => 'react-native',
+                'name' => 'React Native',
+                'version' => '0.3.0',
+                'url' => 'https://github.com/appwrite/sdk-for-react-native',
+                'package' => 'https://npmjs.com/package/react-native-appwrite',
+                'enabled' => true,
+                'beta' => false,
```
Comment (abnegate) [inline app/config/platforms.php:145]: Should we start with beta?

**Pull request merged**